Packs/Cisco-umbrella/Integrations/CiscoUmbrellaEnforcement/CiscoUmbrellaEnforcement.py

```diff
@@ -0,0 +1,197 @@
+import demistomock as demisto
+from CommonServerPython import *
+
+''' IMPORTS '''
+
+# Disable insecure warnings
+requests.packages.urllib3.disable_warnings()
+
+
+class Client(BaseClient):
+    """
+    Client to use in the Cisco Umbrella Enforcement integration. Overrides BaseClient
+    """
+
+    def __init__(self, base_url, verify, proxy, api_key):
+        super().__init__(base_url=base_url, verify=verify, proxy=proxy, ok_codes=(200, 202, 204))
+        self.api_key = api_key
+
+    def get_domains_list(self, page: str = '', limit: str = '') -> list:
+        domains_list: list = []
+        response = self.get_domain_request(
+            f'{self._base_url}domains?customerKey={self.api_key}&{prepare_suffix(page=page, limit=limit)}')
+        while response and len(domains_list) < int(limit):
+            response_data = response.get('data', [])
+            for domain in response_data:
+                domain.pop('lastSeenAt')
+                domain['IsDeleted'] = False
+                domains_list.append(domain)
+            response_next_page = response.get('meta', {}).get('next', '')
+            response = self.get_domain_request(response_next_page) if response_next_page else {}
```

Comment:
for clarity reasons:
```suggestion

            if not response_next_page:
                break
            else:
                 response = self.get_domain_request(response_next_page)
```

Reply:
my indent is probably off